stdpopsim/engines.py

```diff
@@ -12,7 +12,6 @@
 def register_engine(engine):
     """
     Registers the specified simulation engine.
-
```

Comment:
A whole bunch of line deletions seem to have snuck into this diff. I think this is why the docs are failing to build. It's fairly normal to have a single line docs summary at the start of docstring, and then the rest of the docs are separated by a blank line (https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).

Reply:
Yeah, that's strange.. `black` seemed to be unhappy with those blank lines and removed them. Might have to manually tinker with things to solve that issue.